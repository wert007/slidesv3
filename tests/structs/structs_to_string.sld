struct A {
    field: B;
}

struct B {
    a: int;
    b: int;
    c: bool;
}

func main() {
    let a = new A(new B(1, 2, true));
    print(a);
}