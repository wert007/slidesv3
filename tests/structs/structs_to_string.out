struct A {
    field = struct B {
        a = 1,
        b = 2,
        c = true,
    },
}
= 0
